src/Kernel.php

```diff
@@ -33,46 +33,46 @@ class Kernel extends BaseKernel
 {
     use MicroKernelTrait;
 
-    const CONFIG_EXTS = '.{php,xml,yaml,yml}';
+    private const CONFIG_EXTS = '.{php,xml,yaml,yml}';
 
     public function getCacheDir()
```

Comment:
You can add a return type hint here as well. :)

Reply:
Added